WiserTaskScheduler/HelperLibrary/GclServicesHelper.cs

```diff
@@ -0,0 +1,50 @@
+﻿using GeeksCoreLibrary.Core.Interfaces;
+using GeeksCoreLibrary.Core.Models;
+using GeeksCoreLibrary.Core.Services;
+using GeeksCoreLibrary.Modules.Communication.Interfaces;
+using GeeksCoreLibrary.Modules.Communication.Services;
+using GeeksCoreLibrary.Modules.Databases.Interfaces;
+using GeeksCoreLibrary.Modules.GclReplacements.Interfaces;
+using GeeksCoreLibrary.Modules.Objects.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace HelperLibrary;
+
+public class GclServicesHelper
+{
+    /// <summary>
+    /// Get a <see cref="IWiserItemsService"/>.
+    /// Wiser Items Service requires dependency injection that results in the need of MVC services that are unavailable.
+    /// Get all other services and create the Wiser Items Service with one of the services missing.
+    /// </summary>
+    /// <param name="scope">The scope to use.</param>
+    /// <param name="databaseConnection">The database connection to use.</param>
+    /// <param name="gclSettings">Optionally GCL settings, if not provided they will be injected.</param>
+    /// <returns>Returns and instance of <see cref="ICommunicationsService"/></returns>
```

Comment:
Deze method geeft een `IWiserItemsService` terug, dus dit stukje klopt niet. Daarnaast staat er "**and** instance" en dat moet denk ik "**an** instance" zijn.

Reply:
Verbetering doorgevoerd.